### DISCUSSION TITLE
Save Button Should Not Discard Changes When User Sign Up!

### DISCUSSION THREAD

**Discussion by OmarLajam** [bootstrap-email/bootstrap-email#214] [Ideas]
Hi,

When an unauthenticated user wants to save changes made on the editor by clicking on the save button on the top bar, the user is prompted to login or sign up with an email to be able to save changes. 

However, after the sign up completes, all of the changes made on the editor are discarded and can never be restored again. This can be a work of hours!

There should be some way to warn the user before be directed to the sign up page, or to save the changes correctly.

Thanks

**Comment by stuyam** [comment]
This is great feedback, this is now fixed! 🎉  It will preserve the document between sign in process.
